handbook/people/how-we-work.md

```diff
@@ -0,0 +1,43 @@
+---
+title: How we work
+description: Our approach to working together.
+---
+
+# How we work
+
+## Remote
+
+As a remote-first company, we meticulously design our processes and tools to optimize remote work. Our commitment to excellence leads us to seek out and hire top talent globally, constrained only by legal considerations, not geographical boundaries.
+
+We entrust you with the responsibility of crafting your ideal work environment and determining your most productive schedule. Our faith in your judgment is unwavering; we believe you'll make choices that best serve both your needs and the company's goals. Rest assured, we stand ready to provide whatever support you require to thrive in your role. Your success is our success, and we're dedicated to empowering you with the flexibility and resources necessary to achieve it.
+
+> [!TIP] GLOBAL INTERNET WITH A NO GLOBAL LEGAL FRAMEWORK
+> While the internet enables global hiring, legal constraints may limit our ability to employ people from certain countries. We strive to overcome these challenges, but can't guarantee employment if we lack legal representation in your location. We'll always be transparent about such limitations during the hiring process.
+
+## Communication
+
+In this setup, it's important to over-communicate, doing so clearly and concisely, and to include context and expectations to minimize back-and-forth. Even though we use [Slack](https://slack.com) as a communication tool, don't expect it to be used synchronously. Avoid pings and don't expect people to respond immediately.
+
+It's also important to feel comfortable having multiple tasks in progress so that you can make progress on something else while waiting for feedback or a response.
```

Comment:
```suggestion
It's also important to feel comfortable having multiple tasks in progress, so that you can make progress on something else while waiting for feedback or a response.
```